### ISSUE TITLE
[Feature Request] Add blacklist support

### ISSUE THREAD

**Issue by Sollex-21412** [Aseman-Land/Cutegram#141]
It would be nice to have this feature.

**Comment by Dyras** [comment]
Blacklist what exactly? People?

**Comment by Sollex-21412** [comment]
Good question. Do u want anybody or anything else?

**Comment by Dyras** [comment]
I was thinking a blacklist for certain sites so that they don't show up when sent in a message to you.

**Comment by Sollex-21412** [comment]
Nope, i mean the contacts. Contacts blacklist.

**Comment by Dyras** [comment]
Yeah I know that's what you meant. I was just dropping suggestions for other uses.

I wish the blacklist didn't work like blocking though. I don't want the other person to know I'm blocking him/her/it.

**Comment by auchri** [comment]
They dont know it

**Comment by Dyras** [comment]
Perfect! Then I'm getting outta here!

*Walks out the door*
